assets/room_type/storage_bay.tres

```diff
@@ -7,8 +7,8 @@ script = ExtResource("1_jq27t")
 id = 3
 name = "Storage Bay"
 price = 50
-powerConsumption = 100
+power_consumption = 100
 capacity = 0.2
-minTiles = 6
-maxTiles = 15
-tilesetId = 8
```

Comment:
Thanks. When we first started, the examples of class variables were in camel case. Maybe it is an old convention or maybe the examples we found we trash :)

Reply:
Haha yeah I figured as much, I remember these being around for a long time. I don't really mind either way as long as we're consistent